src/main/scala/com/databricks/spark/xml/XmlOptions.scala

```diff
@@ -33,6 +33,7 @@ private[xml] class XmlOptions(
   val rowTag = parameters.getOrElse("rowTag", XmlOptions.DEFAULT_ROW_TAG)
   require(rowTag.nonEmpty, "'rowTag' option should not be empty string.")
   val rootTag = parameters.getOrElse("rootTag", XmlOptions.DEFAULT_ROOT_TAG)
+  val header = parameters.getOrElse("header", XmlOptions.DEFAULT_NULL_VALUE)
```

Comment:
I think this is known as the "XML declaration"?
We don't want to use DEFAULT_NULL_VALUE. the default would already be null.